### ISSUE TITLE
Adding scatterplot makes scale bar disappear 

### ISSUE THREAD

**Issue by vzinche** [mobie/mobie-viewer-fiji#563]
Seems like adding any scatterplot removes the scale bar from BDV. And removing the plot doesn't restore it back. 
To reproduce: open platy browser -> view sbem-segmentation cells -> add plot with all the default options

**Comment by tischi** [comment]
Good catch. I asked [here](https://gitter.im/bigdataviewer/bigdataviewer-core)...
